packages/context/src/interceptor-chain.ts

```diff
@@ -0,0 +1,222 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/context
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as debugFactory from 'debug';
+import {BindingFilter} from './binding-filter';
+import {BindingAddress} from './binding-key';
+import {BindingComparator} from './binding-sorter';
+import {Context} from './context';
+import {transformValueOrPromise, ValueOrPromise} from './value-promise';
+
+const debug = debugFactory('loopback:context:interceptor-chain');
+
+/**
+ * Return value for a method invocation
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type InvocationResult = any;
+
+/**
+ * Array of arguments for a method invocation
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type InvocationArgs = any[];
```

Comment:
AFAICT, `InvocationArgs` are not used by the generic chain, they are specific to invocation-interceptors. Please move them back to `packages/context/src/interceptor.ts `.

Reply:
It's now in `invocation.ts`.